Dockerfile

```diff
@@ -1,11 +1,25 @@
-FROM python:3.11-slim-buster
+FROM python:slim
+
+LABEL org.opencontainers.image.source="https://github.com/toddrob99/searcharr"
+LABEL org.opencontainers.image.description="Docker for SEARCHARR"
+LABEL Name=Searcharr Version="v1.2-beta"
```

Comment:
Same Dockerfile is used for beta and prod, so should not have beta in the version.

Reply:
ok